governance/reviews/cloudevents-08-29-2023.md

```diff
@@ -0,0 +1,210 @@
+
+The SDKs do not appear to have any controls on permissions, and as such are likely very out of sync with project roles.
+
+### Maintainer List(s)
+
+CloudEvents' three types of leaders need to be evaluated separately.
+
+#### Admins
+
+The list of Admins is current, and appears well-balanced, with four admins representing four different organizations.  However, admins are listed only by GH handle, with no affiliation, contact information, or history; listing full information on each admin would enhance transparency.  The Admins were set up as roles-for-life, with only one change of admin since the project was founded in 2018.  There are no defined processes for removing an admin for inactivity, nor any expectation that new Admins will be recruited.
+
+Since, at this time one of the four admins (Ken Owens) is no longer active in the project per Devstats and meeting tracking, it would benefit Cloud Events to document a full Admin lifecycle including recruitment, promotion, qualifications, and emeritus status.  Documenting this and following it would help ensure that the project retains a continuously active leadership.
+
+#### Voting Members
+
+In contrast to the admins, voting members are defined by their activity through participation in spec meetings, ensuring that the list of voting members is always relatively current.  This makes it quite an easy route to get involved in the project, and prevents staleness.  The list of voting members is tracked through a [spreadsheet](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit?pli=1#gid=0), which raises some issues around transparency and auditability (for example, who owns that spreadsheet?).  Balance is ensured by voting rules which do not permit employees of the same company from voting more than once.
+
+Materially, the three active admins are also the three most consistent Voting Members.  On average, 5-8 additional members participate in voting meetings.
+
+#### SDK Maintainers
+
+SDK maintainers are not current. See Subprojects above.
+
+### Evolution
+
+<!--- How has the project's governance evolved over time?  Is the project steadily refining/advancing its governance as the project grows and resolves issues? --->
+
+Governance evolution is the observable changes and improvements the project makes to its governance over the project's lifespan. It is expected that changes will occur over the project's life and that such changes are iterative, tested, and adjusted.
+
+Major milestones in the project's governance over time include:
+
+* Governance created with project founding in 2018
+* Overhaul of governance files for joining CNCF in 2020
+
+Areas of potential future development include:
+
+* Complete reorganization of governance documentation for clarity and completeness.
+* Adoption of a full Admin lifecycle that encourages turnover.
```

Comment:
Not sure we want to encourage turnover just for the sake of turnover.

Reply:
You already have turnover, but you haven't acknowledged it.

Reply:
But you seem to be asking for us to encourage it - I don't see value in that. It happens, yes. But why should be try to force it? We document how it CAN happen here: https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md#admins

Reply:
It's a good idea to encourage it, yes.  Not only to people leave, they burn out as well.  If you're not  doing anything  to recruit new admins, then you won't get any, and pretty soon you'll be exercising the line in the governance that says what to do when the last admin leaves.

Reply:
This one feels like a very subjective thing. We have a process that's working, no one is complaining, we have at least 3 admins to cover stuff.... I don't see an issue. Swapping in new folks "just because" could only make things worse because it means we'll get people who might not actually want to do it beyond the initial "fake" prestige they think it'll give them and they'll half-ass it. Then the entire process runs the risk of falling apart because the folks who really do want to do it have been pushed out.